packages/fxa-settings/src/components/FormPhoneNumber/index.stories.tsx

```diff
@@ -6,22 +6,55 @@ import React from 'react';
 import FormPhoneNumber from '.';
 import { withLocalization } from 'fxa-react/lib/storybooks';
 import { Meta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
 import AppLayout from '../AppLayout';
 
 export default {
   title: 'Components/FormPhoneNumber',
   component: FormPhoneNumber,
-  decorators: [withLocalization],
+  decorators: [
+    (Story) => (
+      <AppLayout>
+        <Story />
+      </AppLayout>
+    ),
+    withLocalization,
+  ],
 } as Meta;
 
+const mockSubmit = async (phoneNumber: string) => {
+  action('submitPhoneNumber')(phoneNumber);
```

Comment:
<img src="https://media1.giphy.com/media/v1.Y2lkPTc5MGI3NjExdXEwZHF4YmVoaXIzYnk0eTJxZzZ1cHdxczBhcTJvcmhjMms3YTBjcyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/4LVS9L49GSSkTuoxeb/giphy.webp">

But really - I haven't played with this enough, seems cool!